### PR TITLE
Tutorial for tuling123 bot

### PR DESCRIPTION
**Pull request by shwetalsoni**
Screenshot from 2021-08-14 12-38-23
Screenshot from 2021-08-14 12-38-43
Screenshot from 2021-08-14 12-39-01
Screenshot from 2021-08-14 12-40-53
Screenshot from 2021-08-14 12-41-08
Screenshot from 2021-08-14 12-41-21
Screenshot from 2021-08-14 12-41-38
Screenshot from 2021-08-14 12-42-02
Screenshot from 2021-08-14 12-42-15
Screenshot from 2021-08-14 12-42-27
Screenshot from 2021-08-14 12-42-38
Screenshot from 2021-08-14 12-42-49

### CODE REVIEW

**Comment by shwetalsoni** [comment]
Tag #891

File: docusaurus/docs/examples/professional/tuling123-bot.mdx [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+
+</TabItem>
+</Tabs>
+
+> If you are using WeCom, you can get token from [puppet service wxwork](http://wechaty.js.org/docs/puppet-services/wxwork).
+
+Run the bot using the following command:
+
+```bash
+npx ts-node tuling-bot.js
```
Comment (sbis04) [inline docusaurus/docs/examples/professional/tuling123-bot.mdx:384]: I guess this can be run just by using `node tuling-bot.js`

Comment (shwetalsoni) [reply docusaurus/docs/examples/professional/tuling123-bot.mdx:384]: Yeah, but I have been doing it this way in my previous PR's and was accepted. Should I change it?

File: docusaurus/docs/examples/professional/tuling123-bot.mdx [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+---
+
+<!-- MDX import -->
+import Tabs     from '@theme/Tabs'
+import TabItem  from '@theme/TabItem'
+
+[![Powered by Wechaty](https://img.shields.io/badge/Powered%20By-Wechaty-brightgreen.svg)](https://github.com/Wechaty/wechaty)
+[![JavaScript](https://img.shields.io/badge/%3C%2F%3E-JavaScript-blue.svg)](https://www.javascript.com/)
+
+Tuling bot is a bot which can talk with you using [Tuling123.com](http://www.turingapi.com/)
```
Comment (proudofsimin) [inline docusaurus/docs/examples/professional/tuling123-bot.mdx]:
```suggestion
Tuling bot is a bot which can talk with you using [Tuling123.com](http://www.turingapi.com/).
```

**Review by proudofsimin — APPROVED** [review]

File: docusaurus/docs/examples/professional/tuling123-bot.mdx (hunk 1 above)
Comment (sbis04) [reply docusaurus/docs/examples/professional/tuling123-bot.mdx:384]: Okay, then no need to change, both of them work.

**Review by sbis04 — APPROVED** [review]

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Comment by shwetalsoni** [comment]
Tag #891

**Pull request merged**